Qt6 multimedia is missing the gstreamer dependencies

When using Qt 6.2 the multimedia module is considered an addon, it downloads correctly but it doesn't bring the necessary dependencies (the gstreamer libs) causing a build error:
```
[ 96%] Linking CXX executable musiqt
/usr/bin/ld: warning: libgstpbutils-1.0.so.0, needed by /home/runner/work/musiqt/Qt/6.2.0/gcc_64/lib/libQt6Multimedia.so.6.2.0, not found (try using -rpath or -rpath-link)
/usr/bin/ld: warning: libgstaudio-1.0.so.0, needed by /home/runner/work/musiqt/Qt/6.2.0/gcc_64/lib/libQt6Multimedia.so.6.2.0, not found (try using -rpath or -rpath-link)
/usr/bin/ld: warning: libgstapp-1.0.so.0, needed by /home/runner/work/musiqt/Qt/6.2.0/gcc_64/lib/libQt6Multimedia.so.6.2.0, not found (try using -rpath or -rpath-link)
/usr/bin/ld: warning: libgstallocators-1.0.so.0, needed by /home/runner/work/musiqt/Qt/6.2.0/gcc_64/lib/libQt6Multimedia.so.6.2.0, not found (try using -rpath or -rpath-link)
/usr/bin/ld: warning: libgstgl-1.0.so.0, needed by /home/runner/work/musiqt/Qt/6.2.0/gcc_64/lib/libQt6Multimedia.so.6.2.0, not found (try using -rpath or -rpath-link)
/usr/bin/ld: warning: libgstvideo-1.0.so.0, needed by /home/runner/work/musiqt/Qt/6.2.0/gcc_64/lib/libQt6Multimedia.so.6.2.0, not found (try using -rpath or -rpath-link)
/usr/bin/ld: /home/runner/work/musiqt/Qt/6.2.0/gcc_64/lib/libQt6Multimedia.so.6.2.0: undefined reference to `gst_gl_sync_meta_set_sync_point'
[more undefined references follow]
```

This is the relevant excerpt from the YAML file:
```
jobs:
  build:
    runs-on: ubuntu-latest
    strategy:
      matrix:
        qt-version: [5.15.2, 6.2.0]

    steps:
    - uses: jurplel/install-qt-action@v2
      with:
        version: ${{ matrix.qt-version }}
        modules: 'addons.qtmultimedia'
```

Full logs attached:
[logs_217.zip](https://github.com/jurplel/install-qt-action/files/7931615/logs_217.zip)

--- a/action/src/main.ts
+++ b/action/src/main.ts
@@ -247,6 +247,7 @@ const run = async (): Promise<void> => {
         const dependencies = [
           "build-essential",
           "libgl1-mesa-dev",
+          "libgstreamer-gl1.0-0",
           "libpulse-dev",
           "libxcb-glx0",
           "libxcb-icccm4",

